### ISSUE TITLE
Check value for rate_duration in CALET notices

### ISSUE THREAD

**Issue by lpsinger** [nasa-gcn/gcn-classic-to-json#34]
The reason I chose bkg_dur1 is that the documentation defines it as "the amount of time (units centi-sec) used to determine the background countrate when used for the lc_significance" and the descriptions for rate_duration is "Interval over rate signal to noise ratio calculation". Although thinking about it again I believe bkg_dur1 is being used to calculate the noise, so should I switch to fore_dur?

_Originally posted by @athish-thiru in https://github.com/nasa-gcn/gcn-classic-to-json/pull/20#discussion_r1702237564_

**Comment by athish-thiru** [comment]
According to the source code, foreground duration (triggering interval) is the binning interval and I can't find any keyword in the the core schema that is equivalent to this, so I'm thinking of saving this value as trigger_duration. I can submit a new pull request for this.

**Comment by athish-thiru** [comment]
Additionally, according to the source code, the url is a fake value to be used as a placeholder until a server is set up by the mission (this may be separate Issue).

**Comment by lpsinger** [comment]
> Additionally, according to the source code, the url is a fake value to be used as a placeholder until a server is set up by the mission (this may be separate Issue).

@jracusin, @Vidushi-GitHub, hopefully CALET has set up a real URL by now. Can you ping them?

**Comment by lpsinger** [comment]
> According to the source code, foreground duration (triggering interval) is the binning interval and I can't find any keyword in the the core schema that is equivalent to this, so I'm thinking of saving this value as trigger_duration. I can submit a new pull request for this.

That's not a trigger duration. @Vidushi-GitHub, would you please work up a PR to add an appropriate field to a core schema?

**Comment by Vidushi-GitHub** [comment]
> > Additionally, according to the source code, the url is a fake value to be used as a placeholder until a server is set up by the mission (this may be separate Issue).
> 
> @jracusin, @Vidushi-GitHub, hopefully CALET has set up a real URL by now. Can you ping them?

I am not sure if other instruments are reporting this specific "fore_duration" (described in https://gcn.gsfc.nasa.gov/sock_pkt_def_doc.html), 4 foreground/triggering intervals 0.25s, 0.5s, 1s and 4s. Right now we should go with a specific field, and add it to the core if it's found in more instruments.

**Comment by lpsinger** [comment]
@Vidushi-GitHub, do I understand that your proposed resolution is to leave this field as it is now?

**Comment by athish-thiru** [comment]
Well right now that field is stored as 'rate_duration' which is defined in the core schema but does not accurately represent the value stored in this field. I could change this to 'foreground_duration' i.e. the same term used in the source code.

**Comment by Vidushi-GitHub** [comment]
> @Vidushi-GitHub, do I understand that your proposed resolution is to leave this field as it is now?

Yes correct! It's not conventional way in other gamma-ray instruments. We will add in core schema if we find more such missions using this field.

**Comment by Vidushi-GitHub** [comment]
> > Additionally, according to the source code, the url is a fake value to be used as a placeholder until a server is set up by the mission (this may be separate Issue).
> 
> @jracusin, @Vidushi-GitHub, hopefully CALET has set up a real URL by now. Can you ping them?

@jracusin shall we ping the CALET team, if they have real URLs for the light curve?

**Comment by athish-thiru** [comment]
I found this [link](http://yoshidalab.mydns.jp/cgbm_trigger/flight/) (It seems to be the webpage for one of the PIs of the mission?) for the CALET mission that does seem to have the light curves for the initial missions but they seem to have stop updating it after 2019. Similarly I found that the initial notices on the gcn archive have a functioning lightcurve url but stop being updated around the same time.

**Comment by lpsinger** [comment]
@Vidushi-GitHub, @athish-thiru, please contact the CALET team to check for sure.
